### PR TITLE
VS Code integration

### LINKED ISSUE #66: VS Code + pylint integration with p5
Hey,

I've installed the lib on my system and copied the demo file. It runs well, but in VS Code with pylint it's giving me quite a lot of errors and warnings like:

- E0102:function already defined line 1 (setup function)
- E0102:function already defined line 1 (draw function)
- E0602:Undefined variable 'mouse_is_pressed'
- W0614:Unused import setup from wildcard import
- and lots more

How  can I set this up to play well with VS Code and pylint?
Sorry if this may sound dumb, I'm pretty new to python :)

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
Added vscode and pylinter integration.

Fixes #66

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,15 +10,15 @@
 
 packages = ['p5']
```
[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply setup.py:12]: How can I do that did that by mistake

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "python.linting.pylintArgs": [
```
[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply .vscode/settings.json:2]: Yes. I will just fix that. Really quick 👍🏼

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply setup.py:12]: Should I make a new pr

[the remark this answers is not in the document]
Comment (ziyaointl) [reply setup.py:12]: I think you can still use this one. I'll reply again with some commands you can try.

[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply setup.py:12]: Are the rest of the changes ok?

**Comment by ziyaointl** [comment]
You can use `git rm` to remove a file, and `git checkout <commit> -- <file>` to revert a file to its previous version.
For exmple, to remove `.vscode/settings.json`
```bash
git rm .vscode/settings.json
```
To revert `setup.py` to the version on the previous master.
```bash
git checkout 3e034c256788e39f38f1190923af18c2e6317db9 -- setup.py
```
Once you are done, use `git add` and then `git commit` to commit your changes.

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ziyaointl) [reply setup.py:12]: You may want to capitalize the S in VS Code. Everything else looks good :)

[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply setup.py:12]: Done

**Comment by Andy-Python-Programmer** [comment]
@ziyaointl I am not on my computer right now. Is there a way to do it using the online editor.

**Comment by ziyaointl** [comment]
@Andy-Python-Programmer Sorry, I'm not aware of the ways to do it in the online editor. Feel free to finish this up once you have access to a computer though :)

**Comment by Andy-Python-Programmer** [comment]
Yep 👍🏻. No problem :)

**Comment by Andy-Python-Programmer** [comment]
Hi @ziyaointl I have resolved the conversation 👍🏼

**Comment by ziyaointl** [comment]
Thanks for the great work!

**Comment by ziyaointl** [comment]
Sorry it seems that some old changes got into master nonetheless. I guess instead of doing a `git checkout` one should be doing a `git rebase master`. I'll add a commit on master that corrects this.

**Comment by Andy-Python-Programmer** [comment]
So is there anything that i have to do @ziyaointl

**Comment by ziyaointl** [comment]
@Andy-Python-Programmer I `cherry-pick`ed your old commit on top of master, so everything should be good now. Thanks again for bearing with my rusty git skills :)

**Comment by Andy-Python-Programmer** [comment]
:)

**Pull request merged**